edge-util/src/Microsoft.Azure.Devices.Edge.Util/HttpClientHelper.cs

```diff
@@ -12,14 +12,15 @@ public class HttpClientHelper
         const string HttpsScheme = "https";
         const string UnixScheme = "unix";
 
+        static Lazy<HttpClient> client = new Lazy<HttpClient>(() => new HttpClient(), isThreadSafe: true);
```

Comment:
I'm not a fan of "static", but this is quick fix. Ideally we should convert `HttpClientHelper` from static class into an "injectable" factory, but this is much more involved for this small fix. What do you think?